### DISCUSSION TITLE
Whatsapp Web Automation.

### DISCUSSION THREAD

**Discussion by rahulbansal16** [crxjs/chrome-extension-tools#778] [Show and tell]
The crxjs plugin is awesome. 
It made building chrome extension super easy. 
Here is my extension https://orso.app

There should be a showoff section in the documentation.

**Comment by JoeyDoey** [comment]
Looks great! good luck 🚀🎉

**Comment by jimy011** [comment]
Automating WhatsApp Web tasks empowers users to handle their conversations seamlessly, ensuring prompt message sending, efficient replies to incoming messages, and targeted forwarding of specific messages, all of which contribute to an enhanced messaging experience. Many people use [OG WhatsApp](https://gbnine.com/ogwhatsapp-apk/) for web automation.
